build/ToolsetPackages/project.lock.json

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
@jaredpar It looks like this commit: https://github.com/agocke/roslyn/commit/4c85400a04339442fcb8d463a08bf1051e0948d7 deleted the file.